### ISSUE TITLE
Unable to add applet to panels (Arch)

### ISSUE THREAD

**Issue by WinnerWind** [zalesyc/budgie-media-player-applet#5]
I am unable to add the applet to the panels even after installing using Arch Linux and the guide given. The option exists but I am unable to add it. Will add further details if needed.

**Comment by WinnerWind** [comment]
I thought more information would be necessary : 
I have all the dependencies as mentioned in the readme (budgie, python 3.11, python-pillow, python-requests, and budgie already comes with GTK)
The applet has a named entry in the desktop manager, but when trying to add it, it simply does not add it, nor throw an error message. Every other applet works, but not this one.
TestWin,py shows a blank window when no music is playing, and when music is playing, the window shows the correct art and working media buttons (as expected)
![image](https://github.com/zalesyc/budgie-media-player-applet/assets/71366717/2049d6f9-c795-4ca9-8fe2-811e80fb1e90)

However, it seems that Budgie refuses to make this work.

**Comment by WinnerWind** [comment]
SOLUTION!
If dependencies are not installed after first meson, install them, then proceed to go into the folder where the git was cloned and delete the `build` folder. 
Install dependencies using the following commands
(You should already have Budgie and GTK, I believe python too is pre-installed with every budgie system, so I am omitting those)
`sudo pacman -S python-requests`
`sudo pacman -S python-pillow`

for good measure, run the meson command given.
then run the install command `cd build && sudo ninja install`

Then it should work as intended
Note : deleting the plugin folder in `/usr/lib/budgie-desktop/plugins` is also a good measure, before running the above few commands

Arch Linux x86_64 Budgie 10.9.1 
Hope it helps the next person.

**Comment by zalesyc** [comment]
I am glad that you were able to resolve the issue, I will try to improve the rename to prevent similar future issues.

**Comment by WinnerWind** [comment]
HI there! Just a follow up.
I thought it would be helpful to state *exactly* which package is needed
instead of `python3-requests`, writing `python-requests` reduces confusion. Thank you though :)

**Comment by zalesyc** [comment]
I've just updated the readme, I moved the dependencies section into a dropdown in the install section, to be less confusing, 19e2b27. 

In the now dropdown it's `python3-requests` instead of just `python-requests` because there also is a version for python2, [link](https://pkgs.org/download/python2-requests).

But I still think that your comments are useful, because I am deeply involved in the project, so I don't see the issues from the perspective of an ordinary user.
